apps/imgviewer.c

```diff
@@ -47,22 +49,36 @@ int height = 300;
 sprite_t img = {0};
 
 #define APPLICATION_TITLE "Image Viewer"
+#define IMGVIEWER_VERSION   "1.4.0" 
+#define IMGVIEWER_COPYRIGHT "Copyright 2012-2019 K. Lange <\033[3mklange@toaruos.org\033[23m>"
 
+#define DEBUG 1
```

Comment:
`DEBUG` defines are typically either defined or undefined but not defined to `1`.